content/hardware/04.pro/boards/portenta-x8/features.md

```diff
@@ -1,5 +1,11 @@
 <FeatureDescription>
-  The Portenta X8 is a high performance board, with many exceptional features. With two microprocessors combined, the NXP® i.MX 8M Mini MPU (Linux) and STM32H747XI dual Cortex®-M7+M4 32bit low power ARM® MCU (Arduino), this board is a great source to power the upcoming generation of Industrial Internet of Things.
+
+The Portenta X8 is a high-performance board, with many exceptional features. With two microprocessors combined, the NXP® i.MX 8M Mini MPU (Linux) and STM32H747XI dual Cortex®-M7+M4 32bit low power ARM® MCU (Arduino), this board is a great source to power the upcoming generation of the Industrial Internet of Things.  
+
+Thanks to the use of containers, the Portenta X8 offers an easy and secure way to implement applications running in a controlled environment. The user has full control of the containers that the board is executing, having the option of creating and running custom containers on its own without the requirement of any additional subscription services and totally free of charge.  
+
+In case the user needs a more advanced and scalable professional solution, Arduino has created, in collaboration with Foundries, a subscription service designed for professional solutions with many advanced features like updates Over-The-Air and full control of the containers and management of multiple boards. Learn more about an Arduino Pro Subscription [here](https://cloud.arduino.cc/plans/).
```

Comment:
I am just rephrasing it a little bit and adding the right link: 

In case the user needs a more advanced and scalable professional solution, Arduino has created, in collaboration with Foundries, a subscription service with many enterprise-grade features: 
- Secure maintenance of Portenta X8 Linux distribution and applications over time
- Portenta X8 fleets monitoring from anywhere
- Secure Over-The-Air updates to target Portenta X8 devices/fleets 

Learn more about the so-called Portenta X8 Manager [here](https://cloud.arduino.cc/plans#business).

Reply:
replacing a , with ; as being a short pause